update_external_versions.conf

```diff
@@ -78,12 +82,62 @@
 # in the file paths (like on arti.dev), the type parameter should be omitted entirely, otherwise
 # no images will be found.
 
-# We actually want the latest stable RPM version of csm-ssh-keys, but that function is not yet
-# available in update_external_versions. Fortunately, however, we use the same version for
-# all artifacts in CMS repos (plus or minus the build tag that is appended to unstable docker
-# and helm charts). Because of this, we will get the latest stable csm-ssh-keys docker version,
-# and just strip off the build tag if one is present, and append "-1" to the end. That is the
-# latest stable RPM version.
-image: csm-ssh-keys
+#########################################
+# Form 2: RPMs
+#########################################
+
+#rpm:  RPM_name
+#    major: major number
+#    minor: minor number
+#    outfile: target filename for version
+#    namefile: target filename for RPM filename
+#    urlfile: target filename for full URL to RPM
+#    server: algol60
+#    type: build type
+#    uri: algol60 URI
```

Comment:
You don't enunciate here or below exactly what the URI field does. Presumably, it is the RPM's location in algol60.